### ISSUE TITLE
Data on HF?

### ISSUE THREAD

**Issue by KasperGroesLudvigsen** [meshachaderele/ddsc-llm#2]
Hey, do you want to put the data that you create with this notebook on HuggingFace? :) That way we can easily download it (and you get to share it with others :) ) 
https://github.com/meshachaderele/ddsc-llm/blob/main/process.ipynb

**Comment by meshachaderele** [comment]
Yes. I will put it on HugginFace. Let me know how to best to name it when I do so.

**Comment by KasperGroesLudvigsen** [comment]
Hm, good question. Maybe something that indicates the original name of the dataset?

**Comment by meshachaderele** [comment]
> Hey, do you want to put the data that you create with this notebook on HuggingFace? :) That way we can easily download it (and you get to share it with others :) ) https://github.com/meshachaderele/ddsc-llm/blob/main/process.ipynb

I have the dataset on HF now.
https://huggingface.co/datasets/meshachaderele/negative-positive-wikipedia-2023-11-da

**Comment by meshachaderele** [comment]
I will try the vllm you suggested on the dataset to generate the query.

**Comment by KasperGroesLudvigsen** [comment]
That's actually what I am going to commit in a sec :)

**Comment by KasperGroesLudvigsen** [comment]
I just pushed to the branch "kasper". It works nicely with vllm and even the 8b model produces reasonable responses. Am going to do a vibe check with expanse 32b

https://github.com/meshachaderele/ddsc-llm/tree/kasper

**Comment by KasperGroesLudvigsen** [comment]
It's still WiP tho..
